src/data/nat/basic.lean

```diff
@@ -1121,6 +1121,16 @@ end
 theorem fact_mul_fact_dvd_fact {n k : ℕ} (hk : k ≤ n) : fact k * fact (n - k) ∣ fact n :=
 by rw [←choose_mul_fact_mul_fact hk, mul_assoc]; exact dvd_mul_left _ _
 
+lemma choose_symm {n k : ℕ} (hk : k ≤ n) : choose n k = choose n (n-k) :=
```

Comment:
I think if you swap the sides of the equality, then it is written in simp-direction. You might even make it a simp-lemma then.

Reply:
Thanks for the suggestion!